Submissions/AvdheshKumarAKG/Assignment1/BC-Assignment1.py

```diff
@@ -0,0 +1,58 @@
+from hashlib import sha256
+import time
+import hashlib
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+
+		self.timestamp = timestamp
+		self.data=data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+
+		HASHobj = hashlib.sha256()
```

Comment:
are you sure you checked it before submitting?

it should not work, as you have imported sha256 from hashlib(see line 1), so can only do the following

```HASHobj = sha256()```